v2.0/getting-started/kubernetes/installation/aws.md

```diff
@@ -278,23 +278,32 @@ Verify that you can access the Kubernetes API.  The following command should ret
 ```shell
 ./kubectl get nodes
 ```
-### 4.2 Configure Outbound NAT
+
+## 5. Install Addons
+
+{% include {{page.version}}/install-k8s-addons.md %}
+
+## 6. Configure the Calico IP pool
```

Comment:
We've already disabled Source/Dest checks. Why not point out this is only required if going Cross VPC, and link to the new AWS guide for an explanation?